ecommerce/extensions/payment/exceptions.py

```diff
@@ -25,3 +25,8 @@ class InvalidCybersourceDecision(GatewayError):
 class PartialAuthorizationError(PaymentError):
     """The amount authorized by the payment processor differs from the requested amount."""
     pass
+
+
+class InvalidPayUDecision(GatewayError):
```

Comment:
Can we place this into the payu.py file. That way the complete payment processor is contained in its file